drivers/soundwire/intel.c

```diff
@@ -283,6 +320,12 @@ static void intel_debugfs_init(struct sdw_intel *sdw)
 	debugfs_create_file("intel-registers", 0400, sdw->debugfs, sdw,
 			    &intel_reg_fops);
 
+	debugfs_create_file("intel-m-datamode", 0200, sdw->debugfs, sdw,
+			    &intel_set_m_datamode_fops);
+
+	debugfs_create_file("intel-s-datamode", 0200, sdw->debugfs, sdw,
+			    &intel_set_s_datamode_fops);
+
```

Comment:
Should we allow user get current data mode? ie. Provide _get function as well.

Reply:
Sorry I missed this comment earlier. In general we don't provide get_ functions, the user explicitly changes the value so they should know. Adding a get does not really bring any added value here, the kernel never changes the value on its own.